src/components/AppLoader/getDateFnLocale.ts

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable complexity */
+
+// Vite struggles with dynamic import paths, and in our case import(`date-fns/locale/${locale}/index.js`) throws an error.
+// Vite’s recommended solution is import.meta.glob, but using it on node_modules would significantly bloat the bundle, so it’s best to avoid it.
+// Therefore, I opt to use a switch/case with static imports to load only the locale we need.
```

Comment:
Clever! Might be nice if Kai was aware of this - I know there are many more apps using data-fns.